source/contributing/implementation/debugging/mixin.rst

```diff
@@ -0,0 +1,9 @@
+===============
+Debugging Mixin
+===============
+
+.. note::
```

Comment:
don't really wanna duplicate the mixin wiki, but maybe some useful stuff like the java arguments to export mixin in production https://github.com/SpongePowered/Mixin/wiki/Mixin-Java-System-Properties

not sure what this page could contain otherwise, maybe some coredevs have any other pointers what they use commonly to debug mixins that isnt documented already

Reply:
My idea for this page is to address ST-DDT's [comments](https://github.com/SpongePowered/SpongeDocs/pull/661#issuecomment-364664272):

> Debugging options such as the post mixin class content output (I forgot the command line option for that)
> Tools for decompiling the post mixing code (jd-gui or similar ones)
> Common issues you might encounter (**Broken mixin**, NPEs, PhaseErrors with suggestions how to fix them or a link to their respective documentation)
> Help with finding the error in your mixin based on the stacktrace (The only help you get is the line number)

and the link you mentioned.

Reply:
Likewise, if you use MinecraftDev plugin for IntelliJ, you can debug and step through mixin code without having to deal with outputted class files from the mixin class loader.

I can do a demonstration gif of sorts stepping through sponges changes with both SpongeCommon and SpongeForge intersecting each other.